test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -2196,6 +2196,60 @@ TEST_F(AtenXlaTensorTest, TestOneIndexTransfer) {
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestCount_Nonzero_nodim) {
+  torch::Tensor a = torch::zeros({3, 3}, torch::TensorOptions(torch::kFloat));
+  a[0][1] = 1.0;
+  a[0][2] = 1.0;
+  a[2][2] = 1.0;
+  torch::Tensor b = torch::count_nonzero(a);
+  ForEachDevice([&](const torch::Device& device) {
+    torch::Tensor xla_a = CopyToDevice(a, device);
+    torch::Tensor xla_b = torch::count_nonzero(xla_a);
+    AllClose(b, torch::_cast_Long(xla_b));
+
+    ExpectCounterChanged("xla::count_nonzero", cpp_test::GetIgnoredCounters());
+    ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
```

Comment:
nit, move this outside the `ForEachDevice`, I don't think you need to call `ReetCounters`?

Reply:
done